.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+**Description:**
+
+**Testing Instructions:**
+
+**Related Issue:**
```

Comment:
Should we put a `#` or `Fixes #` here to push them in the right direction?

Reply:
Should we also suggest including a link to Zulip, if applicable? If so, might it make more sense like this?

```
**Related Links**

* Fixes #(github-issue)
* Zulip:
```

Reply:
I had thought about including something like `#` or `Fixes #` in the template, but I didn't love having info that possibly needed to be deleted. The link on "Link the issue" in the best practices document does provide documentation on what keywords you can use to link the issue, so I had hoped that that would be sufficient. Maybe if we do want to include the Zulip link, it is worth calling out both even if they sometimes need to be deleted.

Reply:
OK. Yeah, that's a good point. Zulip belongs more in the issue than the PR. And if we only have one, then we don't need the bulleted format. And I had commented this before I saw the link in the BP document. So... I'm fine with just leaving it as-is!